parsons/actblue/actblue.py

```diff
@@ -46,6 +52,8 @@ def __init__(self, actblue_client_uuid=None, actblue_client_secret=None, actblue
             auth=(self.actblue_client_uuid, self.actblue_client_secret),
             headers=self.headers,
         )
+        self.max_retries = check_env.check("ACTBLUE_MAX_RETRIES", max_retries, optional=True)
```

Comment:
In the docstring you're adding, worth mentioning that we'll check for an environmental variable `ACTBLUE_MAX_RETRIES` and if we find it set, it overrides the `max_retries` passed in.